Mathlib/CategoryTheory/Monoidal/Yoneda.lean

```diff
@@ -37,17 +37,15 @@ def Mon_Class.ofRepresentableBy (F : Cᵒᵖ ⥤ MonCat.{w}) (α : (F ⋙ forget
     simp only [← ConcreteCategory.forget_map_eq_coe, ← Functor.comp_map, ← α.homEquiv_comp]
     simp only [whiskerRight_fst, whiskerRight_snd]
     simp only [α.homEquiv_comp, Equiv.apply_symm_apply]
```

Comment:
Any chance the terminal simp can now swallow a few of these?

Reply:
I just tried. No chance.